components/datetime/src/skeleton.rs

```diff
@@ -630,6 +650,26 @@ fn group_fields_by_type(fields: &[Field]) -> FieldsByType {
     FieldsByType { date, time }
 }
 
+/// Alters given Pattern so that its fields have the same length as 'fields'.
+///
+///  For example the "d MMM y" pattern will be changed to "d MMMM y" given fields ["y", "MMMM", "d"].
+fn adjust_pattern_field_lengths(fields: &[Field], pattern: &mut Pattern) {
+    pattern.items_mut().iter_mut().for_each(|item| {
```

Comment:
nit: i'd just `for item in patterns.items_mut() {`

Reply:
Done